tests/pages/catalog/packages/package/PackageTab-cy.js

```diff
@@ -44,53 +41,57 @@ describe("Package Detail Tab", function() {
         );
     });
 
+    it("contains _blank target for links in description", function() {
+      cy.get(".pod.pod-shorter.flush-top.flush-right.flush-left")
```

Comment:
Nit: Do you think we can use get on something with more meaning? I'm not sure if `contains("Description")` would work.